Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -20,6 +20,10 @@
     '2021PU',
     '2021Design',
     '2021DesignPU',
+    '2021HLT',
```

Comment:
We should not change the order here. New workflow should go at the end, i.e. after 2024.

Reply:
OK, will append it to the end.